rDNA/rDNA/tests/testthat/test-phasetransitions.R

```diff
@@ -0,0 +1,46 @@
+context("Testing dna_phaseTransitions")
+
+# Initialize DNA and sample database
+suppressPackageStartupMessages(library("rDNA"))
+dna_init()
+
+# example 1: testing with default values
+test_that("dna_phaseTransitions produces expected output with default settings", {
+  samp <- dna_sample()
+  dna_openDatabase(samp, coderId = 1, coderPassword = "sample")
+  result <- dna_phaseTransitions()
+  expect_s3_class(result, "dna_phaseTransitions")
+  expect_true(!is.null(result$states))
+  expect_true(!is.null(result$modularity))
+  expect_true(!is.null(result$clusterMethod))
+  expect_true(!is.null(result$distmat))
+  dna_closeDatabase()
+  unlink(samp)
+})
+
+# example 2: testing plotting function
+context("Testing autoplot.dna_phaseTransitions")
+
+
+set_up_test_data <- function() {
```

Comment:
Is it possible to use this setup also for the example above? (You might have to move it up for this.) Then you can save a few lines of code up there.